Tasks/ServiceFabricUpdateManifestsV2/Update-DockerImageSettings.psm1

```diff
@@ -41,7 +72,18 @@ function Update-DockerImageSettings
             {
                 if ($codePackage.EntryPoint -and $codePackage.EntryPoint.ContainerHost -and $codePackage.EntryPoint.ContainerHost.ImageName)
                 {
-                    $digest = $imageNameToDigestMapping[$codePackage.EntryPoint.ContainerHost.ImageName]
+                    $imageName = $codePackage.EntryPoint.ContainerHost.ImageName
+                    if ($imageNames -eq $null)
+                    {
+                        # If we don't have the list of images names with tags to map to remove the tag
+                        $tagSeparatorIndex = $imageName.IndexOf(":")
+                        if ($tagSeparatorIndex -ge 0)
+                        {
+                            $imageName = $imageName.Substring(0, $tagSeparatorIndex)
+                        }
+                    }
+
+                    $digest = $imageNameToDigestMapping[$imageName]
                     if ($digest)
```

Comment:
if digest is false, should print a warning that no digest was found for this image name

Reply:
The user doesn't need to build all images locally. For instance you could pull in the redis image to add a redis cache to your cluster.

Reply:
Got it. In that case please just log as message and not warning. This will help in debugging if we are missing replacing a image which should have been.